src/components/buttonnavigation/ButtonNavigationCSS.js

```diff
@@ -0,0 +1,31 @@
+import { colors } from "../colors/index";
+
+export const ButtonNavigationCSS = `
+    position: absolute;
+    padding: 0;
+    background: none;
+    border: none;
+
+    font-family: 'HK Grotesk', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 1.5rem;
+    line-height: 48px;
+    /* identical to box height, or 200% */
+
+    display: flex;
+    align-items: center;
+    text-transform: capitalize;
```

Comment:
not sure if we want to change this so that it's consistent with the other implementations - @ajo01 Also wondering why it's capitalized?
```suggestion
    display: table;
    align-items: center;
```

Reply:
Oh I think I thought it was capitalized before, i copied this code from when I was first working on it few weeks ago when we were given the issues, and forgot to remove it